examples/seismic/model.py

```diff
@@ -66,15 +67,15 @@ class GenericModel(object):
     General model class with common properties
     """
     def __init__(self, origin, spacing, shape, space_order, nbl=20,
-                 dtype=np.float32, subdomains=(), damp_mask=False,
-                 **kwargs):
+                 dtype=np.float32, subdomains=(), bcs="damp",
+                 grid=None):
```

Comment:
nitpicking: I think can fit on the line above

Reply:
fixed